src/utils/images.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Util method to retrieve image details from /images folder from the relative file path,
+ * e.g. "/images/album%/picture$.jpg" -> { imageDirectory: "images%2Falbum%24", fileName: "picture%24.jpg" }
+ */
+export const getImageDetails = (imageLink: string) => {
```

Comment:
nit: we can add return types since this is a `ts` file
```suggestion
export const getImageDetails = (imageLink: string): { fileName: string; imageDirectory: string } => {
```